### PR TITLE
Minor Updates to BIP-119

### PR DESCRIPTION
**Pull request by JeremyRubin**
This PR contains some minor updates to BIP-119 that clarify

1) Deployment params
2) CTV v.s. Anyprevout
3) Synergies with other softforks
4) The choice of SHA-256 v.s. RIPEMD
5) The choice of Untagged hash vs tagged

### CODE REVIEW

**Comment by JeremyRubin** [comment]
cc @ajtowns

File: bip-0119.mediawiki [hunk 1]
```diff
@@ -210,16 +216,24 @@ A PayToBareDefaultCheckTemplateVerifyHash output matches the following template:
...
+Deployment should be done via BIP 9 VersionBits deployed through Speedy Trial.
 
 The start time and bit in the implementation are currently set to bit 5 and
-March 1st, 2020, but this is subject to change while the BIP is a draft.
+NEVER_ACTIVE/NO_TIMEOUT, but this is subject to change while the BIP is a draft.
 
-For the avoidance of unclarity, the parameters are:
+For the avoidance of unclarity, the parameters to be determined are:
 
+     // Deployment of CTV (BIP 119)
```
Comment (kallewoof) [inline bip-0119.mediawiki:226]: Indentation is off

File: bip-0119.mediawiki [hunk 2]
```diff
@@ -529,10 +631,15 @@ for an OP_NOP are a soft fork, so existing software will be fully functional wit
...
 
+In contrast to previous forks, OP_CHECKTEMPLATEVERIFY will not make scripts
+valid for policy until the new rule is active.
+
 Older wallet software will be able to accept spends from OP_CHECKTEMPLATEVERIFY outputs, but will
 require an upgrade in order to treat PayToBareDefaultCheckTemplateVerifyHash chains with a confirmed ancestor as
 being "trusted" (i.e., eligible for spending before the transaction is confirmed).
 
+
+
```
Comment (kallewoof) [inline bip-0119.mediawiki:642]: Don't think you need 3 newlines here.

**Review by kallewoof — COMMENTED** [review]
Some nits. Feel free to ignore.

File: bip-0119.mediawiki (hunk 2 above)
Comment (JeremyRubin) [reply bip-0119.mediawiki]: fixed these both in a followup commit rather than fixup as i noticed some other extra/missing whitespace.

**Pull request merged**